src/Xive/Comb/RamComb.cs

```diff
@@ -98,9 +100,38 @@ public IXocument Xocument(string name)
             return this.xocumentWrap(new CellXocument(Cell(name), name));
         }
 
-        public ICell Cell(string name)
+        public ICell Cell(string cell)
```

Comment:
Change back to name.
Method name already tells us that we get a cell.
Variable reflects what to define to get it.